### PR TITLE
[DONE] Translation of tools-routing.md .. 

### PR DESCRIPTION
**Pull request by 3imed-jaberi**

### CODE REVIEW

File: content/community/nav.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@
     - id: starter-kits
       title: أدوات البداية
     - id: routing
-      title: المسارات
+      title: التوجيه
```
Comment (Aissaoui-Ahmed) [inline content/community/nav.yml]: أعتقد أن لا نغيرها فالتوجية قد يقصد به 
direction

File: content/community/tools-routing.md [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
+* **[component-router](https://github.com/in-flux/component-router):** حل يرتكز على Flux لمسارات المكونات.
+* **[Director](https://github.com/flatiron/director)** - موجه URL صغير و متماثل لجافا سكريبت.
+* **[Finch](http://stoodder.github.io/finchjs/)** - مكتبة بسيطة لمعالجة مسار جافا سكريبت لكنها قوية.
+* **[mvc-router](https://github.com/rajeev-k/mvc-router)** استخدم نموذج MVC لإنشاء تطبيقات React.
+* **[Reach Router](https://reach.tech/router)** موجه سهل الوصول لـ React.
+* **[react-mini-router](https://github.com/larrymyers/react-mini-router)**موجه URL صغير.
+* **[react-passage](https://github.com/dollarshaveclub/react-passage):** يساعد عند الربط أو إعادة التوجيه إلى مسارات قد تكون أو لا تكون في تطبيق React  الخاص بك.
+* **[react-router](https://github.com/rackt/react-router)** - موجه مشهور لـ React.
 * **[react-router-component](https://github.com/andreypopp/react-router-component)** Declarative routing.
```
Comment (Aissaoui-Ahmed) [inline content/community/tools-routing.md]: oops ! 😋

Comment (3imed-jaberi) [reply content/community/tools-routing.md]: 🤣 🤣  ... 
اقترح ترجمة لهذه ههه

File: content/community/nav.yml (hunk 1 above)
Comment (3imed-jaberi) [reply content/community/nav.yml]: @iRayan7

Comment (Fcmam5) [reply content/community/nav.yml]: > أعتقد أن لا نغيرها فالتوجية قد يقصد به
> direction

IMO, the translation is a bit confusing, but I think that the current convention & what @iRayan7 suggested is more correct, since a `router` in Arabic is `المُسَيّر` or `المُوَجّه`, since it's responsible of managing "the choice of the direction (the route)" so it's a  `مُوجّه`. The word `مسار` is closer to `path` (and it's used like that in the majority of the Arabic translations).

TLDR; I support the current convention, we should keep `المُوَجّه` as a translation for `router`.

```suggestion
      title: الموَجّهات
```
Or (as it is :smile: )
```suggestion
      title: التوجيه
```

File: content/community/tools-routing.md [hunk 3]
```diff
@@ -1,18 +1,19 @@
...
-* **[mvc-router](https://github.com/rajeev-k/mvc-router)** Use the Model-View-Controller (MVC) pattern to build React applications.
-* **[Reach Router](https://reach.tech/router)** An Accessible Router for React
-* **[react-mini-router](https://github.com/larrymyers/react-mini-router)** A minimal URL router mixin.
-* **[react-passage](https://github.com/dollarshaveclub/react-passage):** Passage helps when linking or redirecting to routes that may or may not be in your react app.
-* **[react-router](https://github.com/rackt/react-router)** - A popular declarative router for React
+* **[Aviator](https://github.com/swipely/aviator)** - محول مسارات لواجهات
+الويب تم بناءه من أجل وحدات التطبيقات ذات الصفحة الواحدة. ([مثال](https://gist.github.com/hojberg/9549330)).
+* **[Backbone](https://backbonejs.org/)** - تزود Backbone بنية تطبيقات جافا سكريبت الثقيلة من خلال توفير موديل بربط المفتاح بالقيمة (key-value) و الاحداث الخاصة، ومجموعات مع واجهة برمجة تطبيقات غنية بالدوال القابلة للتعداد ، وواجهات مع معالجة الحدث ، ثم توصيلها جميعًا بالتطبيق الحالي الخاص بك عبر واجهة RESTful JSON.
+* **[component-router](https://github.com/in-flux/component-router):** حل يرتكز على Flux لمسارات المكونات.
```
Comment (Fcmam5) [inline content/community/tools-routing.md]:
```suggestion
* **[component-router](https://github.com/in-flux/component-router):** حل يرتكز على Flux لتوجيه المكونات.
```

File: content/community/tools-routing.md [hunk 4]
```diff
@@ -1,18 +1,19 @@
...
+* **[Director](https://github.com/flatiron/director)** - موجه URL صغير و متماثل لجافا سكريبت.
+* **[Finch](http://stoodder.github.io/finchjs/)** - مكتبة بسيطة لمعالجة مسار جافا سكريبت لكنها قوية.
```
Comment (Fcmam5) [inline content/community/tools-routing.md]: This translation is fine, we just need to replace `route` translation
```suggestion
* **[Finch](http://stoodder.github.io/finchjs/)** - مكتبة بسيطة لمعالجة توجيه جافا سكريبت لكنها قوية.
```

And I suggest another translation if you don't mind :)

```suggestion
* **[Finch](http://stoodder.github.io/finchjs/)** - مكتبة قويّة رغم صغرها لمعالجة التوجيه في جافا سكريبت
```

File: content/community/tools-routing.md (hunk 2 above)
Comment (Fcmam5) [inline content/community/tools-routing.md]: Done :sheep: :sunglasses:  
```suggestion
* **[react-router-component](https://github.com/andreypopp/react-router-component)** توجيه تصريحي (Declarative routing).
```

File: content/community/tools-routing.md [hunk 5]
```diff
@@ -1,18 +1,19 @@
...
+* **[mvc-router](https://github.com/rajeev-k/mvc-router)** استخدم نموذج MVC لإنشاء تطبيقات React.
+* **[Reach Router](https://reach.tech/router)** موجه سهل الوصول لـ React.
+* **[react-mini-router](https://github.com/larrymyers/react-mini-router)**موجه URL صغير.
+* **[react-passage](https://github.com/dollarshaveclub/react-passage):** يساعد عند الربط أو إعادة التوجيه إلى مسارات قد تكون أو لا تكون في تطبيق React  الخاص بك.
```
Comment (Fcmam5) [inline content/community/tools-routing.md]: Here it doesn't need replacement :) since `routes` in the original text is refering to `paths`.

File: content/community/nav.yml (hunk 1 above)
Comment (iRayan7) [reply content/community/nav.yml]: I agree with @Fcmam5. Since this page is for routing libraries, and these libraries are used for `توجيه` the user.

Also, `موجهات` is closer to `routers`. `توجيه` is `routing` 👍

File: content/community/tools-routing.md [hunk 6]
```diff
@@ -1,18 +1,19 @@
...
-* **[Director](https://github.com/flatiron/director)** - A tiny and isomorphic URL router for JavaScript.
-* **[Finch](http://stoodder.github.io/finchjs/)** - A simple, yet powerful, javascript route handling library.
-* **[mvc-router](https://github.com/rajeev-k/mvc-router)** Use the Model-View-Controller (MVC) pattern to build React applications.
-* **[Reach Router](https://reach.tech/router)** An Accessible Router for React
-* **[react-mini-router](https://github.com/larrymyers/react-mini-router)** A minimal URL router mixin.
-* **[react-passage](https://github.com/dollarshaveclub/react-passage):** Passage helps when linking or redirecting to routes that may or may not be in your react app.
-* **[react-router](https://github.com/rackt/react-router)** - A popular declarative router for React
-* **[react-router-component](https://github.com/andreypopp/react-router-component)** Declarative routing.
+* **[Aviator](https://github.com/swipely/aviator)** - محول مسارات لواجهات
+الويب تم بناءه من أجل وحدات التطبيقات ذات الصفحة الواحدة. ([مثال](https://gist.github.com/hojberg/9549330)).
```
Comment (Fcmam5) [inline content/community/tools-routing.md]: A small typo, sorry I missed it :disappointed: 
```suggestion
الويب تم بناؤه من أجل وحدات التطبيقات ذات الصفحة الواحدة. ([مثال](https://gist.github.com/hojberg/9549330)).
```

File: content/community/tools-routing.md [hunk 7]
```diff
@@ -1,18 +1,19 @@
...
+* **[component-router](https://github.com/in-flux/component-router):** حل يرتكز على Flux لتوجيه المكونات.
+* **[Director](https://github.com/flatiron/director)** - موجه URL صغير و متماثل لجافا سكريبت.
+* **[Finch](http://stoodder.github.io/finchjs/)** - مكتبة قويّة رغم صغرها لمعالجة التوجيه في جافا سكريبت
+* **[mvc-router](https://github.com/rajeev-k/mvc-router)** استخدم نموذج MVC لإنشاء تطبيقات React.
+* **[Reach Router](https://reach.tech/router)** موجه سهل الوصول لـ React.
+* **[react-mini-router](https://github.com/larrymyers/react-mini-router)**موجه URL صغير.
+* **[react-passage](https://github.com/dollarshaveclub/react-passage):** يساعد عند الربط أو إعادة التوجيه إلى مسارات قد تكون أو لا تكون في تطبيق React  الخاص بك.
+* **[react-router](https://github.com/rackt/react-router)** - موجه مشهور لـ React.
+* **[react-router-component](https://github.com/andreypopp/react-router-component)** توجيه تصريحي (Declarative routing).
```
Comment (Aissaoui-Ahmed) [inline content/community/tools-routing.md]: المسارات المصرح بها

Comment (Fcmam5) [reply content/community/tools-routing.md]: If you mean `Declarative routing`, that would be wrong since `المسارات المصرح بها` means `Declared paths` or `Authorized paths`.
`Declarative` here means the declarative manner of defining the routes with [react-router-component](https://github.com/andreypopp/react-router-component) and it has nothing to do with permissions/authorizations (you can confirm that by checking the library's documentation).
`Routing` is translated as `التوجيه`, and `path` as `مسار`.
So, I believe that `توجيه تصريحي` is the correct translation, since `with that library, we declare (from "Declarative paradigm") our routes` => `بفضل هذه المكتبة نقوم بالتصريح (من "نمط البرمجة التصريحية") بالوِجهات`

Comment (3imed-jaberi) [reply content/community/tools-routing.md]: @iRayan7  @mohshbool  !!
